docs/spec/slashing/state.md

```diff
@@ -49,3 +50,31 @@ Where:
 * `IndexOffset` is incremented each time the candidate was a bonded validator in a block (and may have signed a precommit or not).
 * `JailedUntil` is set whenever the candidate is jailed due to downtime
 * `SignedBlocksCounter` is a counter kept to avoid unnecessary array reads. `SignedBlocksBitArray.Sum() == SignedBlocksCounter` always.
+
+## Slashing Period
+
+A slashing period is a start and end block height associated with a particular validator,
+within which only the "worst infraction counts": the total amount of slashing for
```

Comment:
The "worst-infraction-counts" is a good principal to outline in the `conceptual-overview.md` doc

Reply:
then it could just be linked here

Reply:
So linked.